librz/analysis/il/analysis_il.c

```diff
@@ -0,0 +1,276 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_analysis.h>
+
+/////////////////////////////////////////////////////////
```

Comment:
are these necessary?

Reply:
As visual separators for me, yes. Not this first one though.